polling_stations/apps/data_collection/management/commands/__init__.py

```diff
@@ -339,11 +366,22 @@ def add_residential_address(self, address_info):
         """
         postcode = re.sub('[^A-Z0-9]', '', address_info['postcode'].upper())
 
+        # generate a unique slug so we can provide a consistent url
+        slug = self.slugify(
```

Comment:
Can this be a `get_slug` method that importers can overwrite?  My thinking here is that we should use UPRNs as slugs wherever possible – they are better unique IDs than this (but we don't always have them)